projects/lib/src/configurator/configurator.component.html

```diff
@@ -1,33 +1,36 @@
 <div #offcanvas class="offcanvas" [ngClass]="{'offcanvas-start': ltr, 'offcanvas-end': !ltr}" tabindex="-1" id="offcanvas" *ngIf="{edited: (edited$ | async)} as obs">
   <!-- HEADER -->
   <div class="offcanvas-header bg-light border-bottom">
-    <h5 class="offcanvas-title" *ngIf="!_showTree && obs.edited as edited">
+    <h5 class="offcanvas-title" *ngIf="!isTree && obs.edited as edited">
       <strong>{{edited.config.id}}</strong>
-      <br/>
       <h6 i18n>(Type: <strong>{{edited.context.templates?.[edited.config.type]?.display || edited.config.type}}</strong>)</h6>
     </h5>
-    <h5 class="offcanvas-title" *ngIf="_showTree" i18n>Configuration Tree</h5>
+    <h5 class="offcanvas-title" *ngIf="isTree" i18n>Configuration Tree</h5>
 
     <button id="right-to-left" type="button" [ngClass]="{'ltr': !ltr}" i18n-uib-tooltip uib-tooltip="Slide to {{ltr? 'right':'left'}}" class="btn btn-sm btn-link ms-auto" (click)="ltr = !ltr">
       <svg-icon key="arrow_forward" width="22px" height="25px"></svg-icon>
     </button>
-    <button type="button" class="btn btn-sm" [ngClass]="_showTree? 'btn-square btn-primary' : 'btn-link'" i18n-uib-tooltip uib-tooltip="{{_showTree? 'Hide' : 'Show'}} configuration tree" (click)="showTree(!_showTree)">
+    <button type="button" class="btn btn-sm" [ngClass]="isTree? 'btn-square btn-primary' : 'btn-link'" i18n-uib-tooltip uib-tooltip="{{isTree? 'Hide' : 'Show'}} configuration tree" (click)="showTree(!isTree)">
       <svg-icon key="tree" width="22px" height="25px"></svg-icon>
     </button>
+    <button class="btn btn-sm" i18n-uib-tooltip uib-tooltip="Global configuration" (click)="showGlobalConfiguration()">
+      <svg-icon key="gear" width="22px" height="25px"></svg-icon>
+    </button>
     <button type="button" class="btn-close text-reset" data-bs-dismiss="offcanvas" aria-label="Close" i18n-aria-label></button>
   </div>
 
+  <!-- BODY -->
   <div #offcanvasBody class="offcanvas-body">
 
-    <ng-container *ngIf="!_showTree && obs.edited as edited">
+    <ng-container *ngIf="!isTree && obs.edited as edited">
       <!-- CONFIGURATOR -->
       <div class="text-muted" *ngIf="edited.context.templates?.[edited.config.type]?.description as description">
         {{description}}
         <hr />
       </div>
 
-      <ng-container
-        *ngTemplateOutlet="configurators[edited.config.type]?.templateRef; context: { $implicit: edited }">
+      <ng-container *ngIf="configurators[edited.config.type]?.templateRef"
```

Comment:
You don't need the `*ngIf`. The previous syntax works because it simply displays nothing if the template is undefined.

Reply:
This is right when Typescript configuration is: `strictTemplate: false`  but I set it to `true` to check potiential typings issues.